Userland/Libraries/LibTest/Randomized/Generator.h

```diff
@@ -49,7 +49,9 @@ inline u32 unsigned_int(u32 max)
         return 0;
 
     u32 random = Test::randomness_source().draw_value(max, [&]() {
-        return AK::get_random_uniform(max + 1);
+        // `clamp` to guard against integer overflow and calling get_random_uniform(0).
+        u32 exclusive_bound = AK::clamp(max + 1, max, NumericLimits<u32>::max());
```

Comment:
@ADKaster I think this makes sense; might have misunderstood you on Discord though. This IMHO implements the "attempt to increment, but if you end up with 0, keep the original value" behaviour that I originally had `AK::max` do.

Reply:
Hm. I guess this is all right. I kind of meant something more like `u32 bound = 1 + AK::clamp(max, 0, NumericLimits::max() - 1)` but :shrug: